src/itdelatrisu/opsu/states/MainMenu.java

```diff
@@ -705,6 +705,7 @@ public void mouseWheelMoved(int newValue) {
 
 	@Override
 	public void keyPressed(int key, char c) {
+		UI.getNotificationManager().sendNotification("ayyyy lmao", Colors.GREEN);
```

Comment:
Whoops, thought I removed it >.< was added it to test the bubble notifs, didn't mean to commit it.